packages/contracts-rfq/contracts/FastBridgeV2.sol

```diff
@@ -444,7 +447,7 @@ contract FastBridgeV2 is Admin, IFastBridgeV2, IFastBridgeV2Errors {
             revert NativeTokenCallValueNotSupported();
         }
         // exclusivityEndTime must be in range (0 .. params.deadline]
```

Comment:
Let's update the comment here to reflect that the range now includes zero.

Reply:
i actually thought it was incorrect before (0 was not technically in range before, but was specified in the comment) - but now happens to be accurate & left it alone.

Reply:
@parodime I meant purely in terms of the exclusive/inclusive range notation: `(0 .. X]` vs `[0 .. X]`

Reply:
i'm blind - thanks